### DISCUSSION TITLE
Getting "Storage Error: Tag prop not found."

### DISCUSSION THREAD

**Discussion by xs2tarunkukreja** [vesoft-inc/nebula#5526] [Q&A]
Hello,

I am playing with Basket Player sample. When I run the command :-
UPDATE VERTEX ON player "player101" \
SET sex = 'M';

I got the "Getting "Storage Error: Tag prop not found."

What is the reason? As I can think, Tag player doesn't have any property named as 'sex'. Am I right?
Is there any way to add or update Dynamic property on a vertex? I am not willing to run "Alter Tag"

Regards,
Tarun

**Comment by QingZ11** [comment]
Right, if you are using the sample dataset, `sex` prop does not exist. Therefore, you cannot use the `UPDATE` operation and should use the `ALTER` operation instead.
